packages/client/README.md

```diff
@@ -694,16 +694,18 @@ Proxy publishing and subscribing are handled on the network overlay level. This
 ```js
 
 // Open publish proxy to multiple nodes on stream
-await publishingClient.openProxyConnections(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)
+await publishingClient.setProxies(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)
 
 // Remove publish proxy to multiple nodes on stream
-await publishingClient.closeProxyConnections(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)
+await publishingClient.setProxies(stream, [], ProxyDirection.PUBLISH)
 
-// Open publish proxy to multiple nodes on stream
-await publishingClient.openProxyConnections(stream, ['0x11111...', '0x22222...'], ProxyDirection.SUBSCRIBE)
+```
 
-// Remove publish proxy to multiple nodes on stream
-await publishingClient.closeProxyConnections(stream, ['0x11111...', '0x22222...'], ProxyDirection.SUBSCRIBE)
+By default the client will attempt to open proxy connections to all of the nodes set in  `setProxies`. If you want to limit the number of connections by setting a fourth parameter. In this approach, if the client is disconnected from one of the nodes it will attempt to connect to another node by random.
```

Comment:
```suggestion
By default the client will attempt to open proxy connections to all of the nodes set in  `setProxies`. You can limit the number of connections by setting the `connectionCount` parameter. In this approach, if the client is disconnected from one of the nodes it will attempt to connect to another node by random.
```